### PR TITLE
make contest extends base.html

### PR DESCRIPTION
**Pull request by Changron**

### CODE REVIEW

File: contest/templates/contest/contest.html [hunk 1]
```diff
@@ -79,32 +56,37 @@ <h1 class="panel-title" style="text-align:center;">Time</h1>
...
+              <td>{{ clarification.name }}</td>
+              <td>{{ clarification.asker }}</td>
+              <td>{{ clarification.content }}</td>
+              <td>{{ clarification.reply }}</td>
+            </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+        <div align="center">
+          <button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#myModal">Clarify</button>
```
Comment (nickelcheng) [inline contest/templates/contest/contest.html]: This line is too long.

**Comment by bruce3557** [comment]
I like this refactor.
Great code and easier to read.

**Comment by bruce3557** [comment]
:+1:

**Comment by nickelcheng** [comment]
This PR is good for me.

**Comment by nickelcheng** [comment]
Come to merge.

**Pull request merged**